src/goSuggest.ts

```diff
@@ -41,10 +41,16 @@ function vscodeKindFromGoCodeClass(kind: string, type: string): vscode.Completio
 
 interface GoCodeSuggestion {
 	class: string;
+	package: string;
 	name: string;
 	type: string;
 }
 
+class ExtendedCompletionItem extends vscode.CompletionItem {
+	package: string;
+	document: vscode.TextDocument;
```

Comment:
Is passing the entire document necessary? Since this is only used for getting the filename, why not store the filename directly in the `ExtendedCompletionItem`

Reply:
Yes, lets just pass the minimum information.